### DISCUSSION TITLE
Handle HTML tags

### DISCUSSION THREAD

**Discussion by Froostx** [rfennell/AzurePipelines#1339] [Q&A]
Hi, 

First of all thank you very much for this plugin, it is really useful.

I'm running into a problem when using markdown's unordered lists:

**-My template:**
![image](https://user-images.githubusercontent.com/91484771/189461348-0e3852ca-5471-46e1-88a4-d7840885ccd7.png)

**-Result:**
![image](https://user-images.githubusercontent.com/91484771/189461366-3612fb02-0e17-4824-b0ee-81162785ccb1.png)

**Expected:**
![image](https://user-images.githubusercontent.com/91484771/189461473-2a0b8759-6b00-4f20-915f-00f98a885292.png)

I'm not sure what is putting that enter after the title.

**Comment by rfennell** [comment]
Sorry I have no idea, some quirk of Handlebars rendering I suspect

**Reply by Froostx** [reply]
No worries, i fixed it using sanitize :)
